nevergrad/optimization/test_optimizerlib.py

```diff
@@ -469,28 +470,41 @@ def test_parallel_es() -> None:
             opt.tell(cand, 1)
 
 
+def get_tests_metamodel(seq: bool = False):
+    tests_metamodel = [
+        (2, 8, 1.0, 120, False),
+        (2, 3, 8.0, 130, True),
+        (5, 1, 1.0, 150, False),
+    ]
+
+    if not os.environ.get("CIRCLECI", False):
+        # Interesting tests removed from CircleCI for flakiness (and we do stats when not on CircleCI):
```

Comment:
I don't really like this even without being only local, because this will get in the way as well. Maybe at least move into another file so that these tests are separated from the rest